RxSugar/ObserverBinding.swift

```diff
@@ -17,7 +17,7 @@ precedence 141
  - parameter observable: Observable that sends events.
  - returns: Disposable object that can be used to unsubscribe the observer.
  */
-@warn_unused_result(message="http://git.io/rxs.ud")
```

Comment:
It's removed because it's the default behavior in Swift 3.0. Now you add a different annotation (@disposableResult) to suppress the behavior